### ISSUE TITLE
use_bias=False or True ? 

### ISSUE THREAD

**Issue by joubertdamien** [jmpap/YOLOV2-Tensorflow-2.0#13]
Hello, 

Is it normal that use_bias=False ? 

Ex:
x = Conv2D(32, (3,3), strides=(1,1), padding='same', name='conv_1', use_bias=False)(input_image)

Cheers

Damien

**Comment by jmpap** [comment]
Hello,

I don't know if this is normal or not. I used the YoloV2 architecture of the  'experiencor/keras-yolo2' repository. This could be a hyper-parameter to test ...

Jean-Michel

**Comment by joubertdamien** [comment]
Thanks for the quick answer. I will raise the question to experiencor.

In Darknet's Github page, this information is not mentioned. https://github.com/AlexeyAB/darknet/blob/master/cfg/yolo-voc.2.0.cfg

For example for the first layer: 

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky 

But the first layer of VGG is declared without mentioning the bias as well, so hard to tell.

**Comment by jmpap** [comment]
This seems to be the choice of experiencor because use_bias = True by default in Keras. Can you please inform me of experiencor's answer?
